src/implementation/fs/InMemoryStorage.ts

```diff
@@ -1,10 +1,8 @@
-import { IKeyValueStorage } from '../../src';
-
 /**
  * Stores objects in memory. It can store any key-value pairs.
  * This is just a placeholder for a proper storage implementation.
  */
-export default class InMemoryStorage implements IKeyValueStorage {
+export default class InMemoryStorage {
```

Comment:
Why remove the interface? I think we still need it as the in memory storage is mostly there to cache the h5p hub info. In other scenarios, there might be a Redis cache or something that implements IKeyValueStorage.